time time_stamp 
uint8 status
float64[7] stateVec
uint8[] image
